File: C#/uri-1005.cs
// URI - 1005

using System;

namespace URI
{
    class Program
    {
        static void Main(string[] args)
        {
            double noteA = double.Parse(Console.ReadLine());
            double noteB = double.Parse(Console.ReadLine());

            Console.WriteLine("MEDIA = " + ((noteA * 3.5 + noteB * 7.5) / 11).ToString("F5"));
        }
    }
}

File: JavaScript-Node/uri-1005.js
// URI - 1005

const readline = require('readline')

const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout
})

rl.on('line', input => {
    const noteA = parseFloat(input)

    rl.on('line', input => {
        const noteB = parseFloat(input)
        const averageGrades = (noteA * 3.5 + noteB * 7.5) / 11

        console.log('MEDIA = ' + averageGrades.toFixed(5))

        rl.close()
    })
})